src/vreg_and_chip_reset/vreg.rs

```diff
@@ -46,32 +46,167 @@ pub type HIZ_R = crate::BitReader;
 #[doc = "Field `HIZ` writer - high impedance mode select  
  0=not in high impedance mode, 1=in high impedance mode"]
 pub type HIZ_W<'a, const O: u8> = crate::BitWriter<'a, VREG_SPEC, O>;
-#[doc = "Field `VSEL` reader - output voltage select  
- 0000 to 0101 - 0.80V  
- 0110 - 0.85V  
- 0111 - 0.90V  
- 1000 - 0.95V  
- 1001 - 1.00V  
- 1010 - 1.05V  
- 1011 - 1.10V (default)  
- 1100 - 1.15V  
- 1101 - 1.20V  
- 1110 - 1.25V  
- 1111 - 1.30V"]
-pub type VSEL_R = crate::FieldReader;
-#[doc = "Field `VSEL` writer - output voltage select  
- 0000 to 0101 - 0.80V  
- 0110 - 0.85V  
- 0111 - 0.90V  
- 1000 - 0.95V  
- 1001 - 1.00V  
- 1010 - 1.05V  
- 1011 - 1.10V (default)  
- 1100 - 1.15V  
- 1101 - 1.20V  
- 1110 - 1.25V  
- 1111 - 1.30V"]
-pub type VSEL_W<'a, const O: u8> = crate::FieldWriter<'a, VREG_SPEC, 4, O>;
+#[doc = "Field `VSEL` reader - Output voltage select for on-chip voltage regulator."]
+pub type VSEL_R = crate::FieldReader<VSEL_A>;
+#[doc = "Output voltage select for on-chip voltage regulator.  
+
+Value on reset: 11"]
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[repr(u8)]
+pub enum VSEL_A {
+    #[doc = "6: 0.85V"]
```

Comment:
Missing a setting for 0.80V here
```suggestion
    #[doc = "5: 0.80V"]
    VOLTAGE0_80 = 5
    #[doc = "6: 0.85V"]
```

Reply:
Thank you, I just added it.